docs/guides/push-based-log-export-to-log-integration-migration-guide.md

```diff
@@ -0,0 +1,276 @@
+---
```

Comment:
Migration guide LGTM. Would seek input from Intel team and also let them know we will wait for their migration guide to reference here.